lib/sequel/plugins/bulk_audit.rb

```diff
@@ -12,22 +12,21 @@ def self.apply(model, opts={})
 
       module SharedMethods
         def model_to_table_map
-          @@model_to_table_map ||= ObjectSpace.each_object(Class).select do |klazz|
+          ObjectSpace.each_object(Sequel::Model::ClassMethods).select do |klazz|
```

Comment:
`Sequel::Plugins::BulkAudit::ClassMethods` can be even better

Reply:
In this case do we need to check if Model has BulkAudit in next line?